apps/elmo_queues/consumers.py

```diff
@@ -0,0 +1,86 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+'''Consume data from pulse and insert it into the database.
+'''
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import logging
+
+from django.conf import settings
+from django.db import connection
+
+from kombu import Queue, Exchange
+from kombu.mixins import ConsumerMixin
+
+from life.models import Forest, Repository
+from pushes import utils
+
+
+HGMO = Queue(
+    f"queue/{settings.PULSE_USER}/hgmo",
+    routing_key="#", expires=settings.PULSE_TTL,
+    exchange=Exchange(
+        "exchange/hgpushes/v2",
+        durable=True, auto_delete=False, type="topic", passive=True,
+    )
+)
+
+
+class ElmoConsumer(ConsumerMixin):
+    def __init__(self, connection):
+        self.connection = connection
+
+    def get_consumers(self, Consumer, channel):
+        return [
+            Consumer([HGMO], callbacks=[self.on_message], accept=['json']),
+        ]
+
+    def on_message(self, body, message):
```

Comment:
I think it'd be best to monitor the queue on pulseguardian. @djmitche, do we have something there that Brian could hook in to?

Reply:
Not on pulseguardian.  Pulse has overall message counts, but not per-queue.  That would take some custom thing.

Reply:
This isn't a bad idea. I do have a mechanism to do this.

Reply:
Dustin I do have per-queue metrics now, I should actually turn them off for everything except this because we aren't using them and they're high cardinality due to churn in queue names.

https://bugzilla.mozilla.org/show_bug.cgi?id=1653606

Reply:
That's cool!